assembl/lib/hash_fs.py

```diff
@@ -0,0 +1,17 @@
+from os import path
+
+from hashfs.hashfs import HashFS
+
+from .config import get
+
+_hashfs = None
+
+
+def get_hashfs():
+    global _hashfs
+    if _hashfs is None:
+        root = get('upload_root', 'var/uploads')
+        if root[0] != '/':
+            root = path.abspath(path.join(__file__, '../../..', root))
```

Comment:
is it not one `...` too many?

Reply:
Ah, may work better with a dirname first.